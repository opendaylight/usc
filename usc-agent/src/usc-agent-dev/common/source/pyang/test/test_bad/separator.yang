module "separator" {
  namespace "urn:test:sep";
  prefix sep; 

  sep:ext2{ } // ok

  sep:ext:separatorRequired; // error: separator required

  extension ext {
    argument test;
    status current;
  }

  extension ext2;

}
