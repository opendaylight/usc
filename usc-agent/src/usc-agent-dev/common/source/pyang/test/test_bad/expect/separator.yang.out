separator.yang:7: error: SYNTAX_ERROR
